feature/contributor/src/main/java/io/github/droidkaigi/confsched2019/contributor/item/ContributorProfileItemHolder.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.droidkaigi.confsched2019.contributor.item
+
+import com.squareup.picasso.Picasso
+import com.xwray.groupie.databinding.BindableItem
+import io.github.droidkaigi.confsched2019.contributor.R
+import io.github.droidkaigi.confsched2019.contributor.bindingmodel.ContributorBindingModel
+import io.github.droidkaigi.confsched2019.contributor.databinding.ItemContributorProfileBinding
+import jp.wasabeef.picasso.transformations.CropCircleTransformation
+
+class ContributorProfileItemHolder(
+    private val bindingModel: ContributorBindingModel,
+    private val onProfileClicked: (profileUrl: String) -> Unit
```

Comment:
You can pass activityActionCreator and use it in the item.
It will become more simple. Because we can remove callback `onProfileClicked `

```kotlin
        binding.root.setOnClickListener {
           activityActionCreator.openUrl(contributor.profileUrl)
        }
```